lockbox-ios/Action/CopyAction.swift

```diff
@@ -44,37 +43,3 @@ extension CopyAction: Equatable {
                 lhs.field == rhs.field
     }
 }
-
-struct CopyConfirmationDisplayAction: Action {
-    let field: CopyField
-}
-
-extension CopyConfirmationDisplayAction: Equatable {
-    static func ==(lhs: CopyConfirmationDisplayAction, rhs: CopyConfirmationDisplayAction) -> Bool {
-        return lhs.field == rhs.field
-    }
-}
-
-class CopyActionHandler: ActionHandler {
-    static let shared = CopyActionHandler()
-
-    private let dispatcher: Dispatcher
-    private let pasteboard: UIPasteboard
-
-    init(dispatcher: Dispatcher = Dispatcher.shared,
-         pasteboard: UIPasteboard = UIPasteboard.general) {
-        self.dispatcher = dispatcher
-        self.pasteboard = pasteboard
-    }
-
-    func invoke(_ action: CopyAction) {
-        let expireDate = Date().addingTimeInterval(TimeInterval(Constant.number.copyExpireTimeSecs))
-
-        self.pasteboard.setItems([[UIPasteboardTypeAutomatic: action.text]],
-                options: [UIPasteboardOption.expirationDate: expireDate])
-
-        self.dispatcher.dispatch(action: CopyConfirmationDisplayAction(field: action.field))
-        // only for telemetry purposes, no one is listening for this
-        self.dispatcher.dispatch(action: action)
```

Comment:
Would Telemetry pick this up automatically now or this should be addressed?

Reply:
Telemetry picks this up automatically now; as long as an `Action` inherits from the `TelemetryAction` protocol and goes through the `Dispatcher`, it will be logged in telemetry.